spg/Animal/bin/barnyard.pl

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Animal;
+use Cow;
+use Horse;
+use Sheep;
+use Mouse;
+
+print "Build your own farm!\n";
+
+my @barnyard
+
+while ( my $input = <STDIN> ) {
+  chomp( $input ); 
+  push @barnyard, $input;
+}
```

Comment:
I don't compile my guesses! But yeah, that's something that's always bugged me about the behaviour of `chomp`. _Is_ the first method faster (not saying it isn't, but it's worth backing efficiency claims up)?